highlevel_instructions/dual_curator.py

```
@@ -106,51 +105,51 @@ async def curate_sources(
+
         except Exception as e:
             logger.error(f"Error in enhanced dual curation: {e}")
             # Fallback to simple curation
             return await self._single_curation(research_data)
-    
+
     async def _single_curation(self, research_data: List[Any]) -> List[Any]:
         """Fallback single curation when dual curation is disabled or fails."""
         try:
             return await self.primary_curator.curate_sources(research_data)
         except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)